Documentation/Browser/AddNewIDLFile.md

```diff
@@ -0,0 +1,31 @@
+# Adding a new IDL file
+
+Serenity's build system does a lot of work of turning the IDL from a Web spec into code, but there are a few things you'll need to do yourself.
+
+For the sake of example, let's say you're wanting to add the `HTMLDetailsElement`.
+
+1. Create `LibWeb/HTML/HTMLDetailsElement.idl` with the contents of the IDL section of the spec. In this case, that would be:
+```
```

Comment:
```suggestion
```webidl
```

Reply:
(for pretty syntax highlighting)

Reply:
Thanks! I didn't know that was supported. :^)